biz.aQute.bndlib/src/aQute/bnd/osgi/Verifier.java

```diff
@@ -84,7 +84,7 @@ public String toString() {
 
 	public final static Pattern	ReservedFileNames				= Pattern
 																		.compile(
-																				"CON(\\..+)?|PRN(\\..+)?|AUX(\\..+)?|CLOCK$|NUL(\\..+)?|COM[1-9](\\..+)?|LPT[1-9](\\..+)?|"
+																				"CON(\\..+)?|PRN(\\..+)?|AUX(\\..+)?|CLOCK\\\\$|NUL(\\..+)?|COM[1-9](\\..+)?|LPT[1-9](\\..+)?|"
 																						+ "\\$Mft|\\$MftMirr|\\$LogFile|\\$Volume|\\$AttrDef|\\$Bitmap|\\$Boot|\\$BadClus|\\$Secure|"
```

Comment:
I guess we need input from @pkriens since there should be consistency here. `CLOCK$` is  file name just like `$Mft`.

The issue may be in the way the test case is written.